packages/ffe-radio-button-react/src/RadioSwitch.js

```diff
@@ -13,9 +13,12 @@ const RadioSwitch = props => {
         rightValue,
         condensed,
         dark,
+        'aria-invalid': ariaInvalid,
         ...rest
     } = props;
 
+    const noneSelected = props.selectedValue === undefined;
```

Comment:
Since we're dealing with javascript, it would be great if every falsy value passed this check. 
So basically, consider `null` and `''` for `selectedValue` to also mean that nothing is selected. Be vary that `0` is something that should be considered a valid selected value though.

Maybe
```javascript
const noneSelected = [null, undefined, ''].includes(props.selectedValue);
```

I think that would reduce the potential for suprises for consumers.